src/parser/jobs/blm/modules/RotationWatchdog.js

```diff
@@ -297,8 +317,36 @@ export default class RotationWatchdog extends Module {
 	}
 
 	_getMissingFire4Count(count, hasConvert) {
-		const NotEnoughUH = this._umbralHeartStacks  < 2
-		const expected = EXPECTED_FIRE4 + (hasConvert ? FIRE4_FROM_CONVERT : 0) - (NotEnoughUH ? 1 : 0)
+		let expected = EXPECTED_FIRE4 + (hasConvert ? FIRE4_FROM_CONVERT : 0)
```

Comment:
I feel like this could be written more cleanly but  it's 12:12AM right now and fucked if I can think of what it'd be.